lib/core-integration/src/Test/Integration/Scenario/API/Shelley/TransactionsNew.hs

```diff
@@ -1803,17 +1803,13 @@ spec = describe "NEW_SHELLEY_TRANSACTIONS" $ do
             ]
 
     it "TRANS_NEW_BALANCE_05/ADP-1286 - \
-        \I can balance correctly in case I need to spend my remaining ADA for fee" $
+        \I can balance correctly in case I have just enough" $
         \ctx -> runResourceT $ do
-        liftIO $ pendingWith
-            "ADP-1286 - ValueNotConservedUTxO: Transaction seems balanced incorrectly \
-            \in case when less than minUtxOValue is left on the wallet"
-        wa <- fixtureWalletWith @n ctx [3_000_000]
-        -- PlutusScenario.pingPong_1 is sending out 2₳ therefore tx fee
-        -- needs to be 1₳ to comply with minUTxOValue constraint
-        let expectedFee = 1_000_000
+        let initialBalance = 3_180_000
+        wa <- fixtureWalletWith @n ctx [initialBalance]
+        -- PlutusScenario.pingPong_1 is sending out 2₳
+        let amt = 2_000_000
         let balancePayload = Json PlutusScenario.pingPong_1
-        let hasExpectedFee = [expectField (#fee . #getQuantity) (`shouldBe` expectedFee)]
```

Comment:
Could revisit as part of this PR 🤔 

When I was talking to Piotr I didn't know paying higher fees when we cannot afford a change output was already the normal behaviour of the old tx workflow.

Reply:
Supporting higher fees seemed to fix the pending tests without many changes 👍 

However, I'd now like to be clearer about:
- Why I needed to tweak the wallet balance in `TRANS_NEW_BALANCE_02c`
- Why there is a slight regression for balancing pingPong_2: wallet now requires 3 ada rather than 2.75 ada

Maybe improving logging will give some insight.

Reply:
> Why I needed to tweak the wallet balance in TRANS_NEW_BALANCE_02c
> Why there is a slight regression for balancing pingPong_2: wallet now requires 3 ada rather than 2.75 ada

After looking at it, the cases for 3 and 3.05 ada make sense to me: https://github.com/input-output-hk/cardano-wallet/pull/3100#discussion_r818642803

I'm not sure what changed, but it might be that the new behaviour is more "correct" as for being conservative, and it's such a small change in behaviour that I wouldn't worry about it.